src/QsCompiler/Tests.Compiler/Tests.Compiler.fsproj

```diff
@@ -175,8 +178,12 @@
     <PropertyGroup>
       <ExecutionTarget>$(MSBuildThisFileDirectory)..\TestTargets\Simulation\Example\bin\$(Configuration)\netcoreapp3.0\Example.dll</ExecutionTarget>
       <LibraryTarget>$(MSBuildThisFileDirectory)..\TestTargets\Libraries\Library1\bin\$(Configuration)\netcoreapp3.0\Library1.dll</LibraryTarget>
+      <LibraryTargetCopy>$(MSBuildThisFileDirectory)..\TestTargets\Libraries\Library1\bin\$(Configuration)\netcoreapp3.0\Library2.dll</LibraryTargetCopy>
```

Comment:
The folder here should be Library2 as well, and we should set the target framework to netcoreapp3.1 as well. Also, it would be nice to rename the property names to LibraryTarget1 and LibraryTarget2 or similar.

Reply:
With the new test, the extra library target is no longer needed, so I removed this line. I think Library1 will get updated to .NET Core 3.1 when master is merged into the feature branch again.